Signum.React/Scripts/Lines/EntityRepeater.tsx

```diff
@@ -14,6 +14,9 @@ export interface EntityRepeaterProps extends EntityListBaseProps {
   createAsLink?: boolean | ((er: EntityRepeaterController) => React.ReactElement<any>);
   avoidFieldSet?: boolean;
   createMessage?: string;
+  getComponentWithIndex?: (ctx: TypeContext<any>, index: number) => React.ReactElement<any>;

```

Comment:
What is this for?

Reply:
this is an alternative to getComponent but the index context is given to the lambda. An example would be if the parent component  should keep a ref to each component